load_sample_data.sh

```diff
@@ -56,35 +58,35 @@ fi
 #######
 
 # Set up API database and upstream
-docker-compose exec -T "$WEB_SERVICE_NAME" bash -c "python3 manage.py migrate --noinput"
+just dc exec -T "$WEB_SERVICE_NAME" python3 manage.py migrate --noinput
+
 # Create a superuser and a user for integration testing
-# Not that the Python code uses 4 spaces for indentation after the tab that is stripped by <<-
-docker-compose exec -T "$WEB_SERVICE_NAME" bash -c "python3 manage.py shell <<-EOF
-	from django.contrib.auth.models import User
-	usernames = ['continuous_integration', 'deploy']
-	for username in usernames:
-	    if User.objects.filter(username=username).exists():
-	        print(f'User {username} already exists')
-	        continue
-	    if username == 'deploy':
-	        user = User.objects.create_superuser(username, f'{username}@example.com', 'deploy')
-	    else:
-	        user = User.objects.create_user(username, f'{username}@example.com', 'deploy')
-	    user.save()
-	EOF"
+echo "
+from django.contrib.auth.models import User
+usernames = ['continuous_integration', 'deploy']
+for username in usernames:
+	if User.objects.filter(username=username).exists():
+		print(f'User {username} already exists')
+		continue
+	if username == 'deploy':
+		user = User.objects.create_superuser(username, f'{username}@example.com', 'deploy')
+	else:
+		user = User.objects.create_user(username, f'{username}@example.com', 'deploy')
+	user.save()
```

Comment:
The Python part here should be indented with 4 spaces, instead of tabs, as per PEP-8.